src/db12/__main__.py

```diff
@@ -0,0 +1,163 @@
+    return result
+
+def multiple_dirac_benchmark_cli(copies, iterations_num, extra_iter):
+    '''Function that calls multiple_dirac_benchmark and prints
+    its results and returns them'''
+    result = multiple_dirac_benchmark(copies, iterations_num, extra_iter)
+    print(
+        result["copies"],
+        result["sum"],
+        result["arithmetic_mean"],
+        result["geometric_mean"],
+        result["median"],
+    )
+    print(" ".join([str(k) for k in result["raw"]]))
+    return result
+
+def wholenode_dirac_benchmark_cli(iterations_num, extra_iter):
+    '''Function that calls wholenode_dirac_benchmark and prints
+    its results and returns them'''
+    result = wholenode_dirac_benchmark(iterations_num, extra_iter)
+    print(
+        result["copies"],
+        result["sum"],
+        result["arithmetic_mean"],
+        result["geometric_mean"],
+        result["median"],
+    )
+    print(" ".join([str(j) for j in result["raw"]]))
+    return result
+
+def main():
+    """Main function"""
+    help_string = """dirac_benchmark.py [--iterations ITERATIONS] [--extra-iteration]
+                  [COPIES|single|wholenode|jobslot|version|help] 
+Uses the functions within dirac_benchmark.py to run the DB12 benchmark from the 
+command line.
+By default one benchmarking iteration is run, in addition to the initial 
+iteration which DB12 runs and ignores to avoid ramp-up effects at the start.
+The number of benchmarking iterations can be increased using the --iterations
+option. Additional iterations which are also ignored can be added with the 
+--extra-iteration option  to avoid tail effects. In this case copies which
+finish early run additional iterations until all the measurements finish.
+The COPIES (ie an integer) argument causes multiple copies of the benchmark to
+be run in parallel. The tokens "wholenode", "jobslot" and "single" can be 
+given instead to use $MACHINEFEATURES/total_cpu, $JOBFEATURES/allocated_cpu, 
+or 1 as the number of copies respectively. If $MACHINEFEATURES/total_cpu is
+not available, then the number of (logical) processors visible to the 
+operating system is used.
+Unless the token "single" is used, the script prints the following results to
+two lines on stdout:
+COPIES SUM ARITHMETIC-MEAN GEOMETRIC-MEAN MEDIAN
+RAW-RESULTS
+The tokens "version" and "help" print information about the script.
+The source code of dirac_benchmark.py provides examples of how the functions
+within dirac_benchmark.py can be used by other Python programs.
+dirac_benchmark.py is distributed from  https://github.com/DIRACGrid/DB12
+"""
+
+    copies = None
+    iterations = 1
+    extra_iteration = False
+
+    parser = argparse.ArgumentParser()
+    #pylint: disable=line-too-long
+    parser.add_argument("--iterations", nargs='?', help="number of iterations to perform", default='')
+    parser.add_argument("--extra-iteration", nargs='?', help="whether an extra iteration is needed", default='')
```

Comment:
```suggestion
    parser.add_argument("--extra-iteration", help="whether an extra iteration is needed", action='store_true'')
```

In this way `extra_iteration` will be `true` if you pass `--extra-iteration`, but if if you do not pass it in argument, it will be `false`.